server/service/logging_queries.go

```diff
@@ -42,3 +43,111 @@ func (mw loggingMiddleware) ApplyQuerySpecs(ctx context.Context, specs []*kolide
 	err = mw.Service.ApplyQuerySpecs(ctx, specs)
 	return err
 }
+
+func (mw loggingMiddleware) ListQueries(ctx context.Context, opt kolide.ListOptions) ([]*kolide.Query, error) {
+	var (
+		loggedInUser = "unauthenticated"
+		err          error
+	)
+	vc, ok := viewer.FromContext(ctx)
+	if ok {
+		loggedInUser = vc.Username()
+	}
+	defer func(begin time.Time) {
+		mw.logger.Log(
+			"method", "ListQueries",
+			"err", err,
+			"user", loggedInUser,
+			"took", time.Since(begin),
+		)
+	}(time.Now())
+	query, err := mw.Service.ListQueries(ctx, opt)
+	return query, err
+}
+func (mw loggingMiddleware) GetQuery(ctx context.Context, id uint) (*kolide.Query, error) {
+	var (
+		loggedInUser = "unauthenticated"
+		err          error
+	)
+	vc, ok := viewer.FromContext(ctx)
+	if ok {
+		loggedInUser = vc.Username()
+	}
+	defer func(begin time.Time) {
+		mw.logger.Log(
+			"method", "GetQuery",
+			"err", err,
+			"user", loggedInUser,
+			"took", time.Since(begin),
+		)
+	}(time.Now())
+	query, err := mw.Service.GetQuery(ctx, id)
+	return query, err
+}
+func (mw loggingMiddleware) NewQuery(ctx context.Context, p kolide.QueryPayload) (*kolide.Query, error) {
+	var (
+		query        *kolide.Query
+		loggedInUser = "unauthenticated"
+		err          error
+	)
+	vc, ok := viewer.FromContext(ctx)
+	if ok {
+		loggedInUser = vc.Username()
+	}
+	defer func(begin time.Time) {
+		mw.logger.Log(
+			"method", "NewQuery",
+			"err", err,
+			"user", loggedInUser,
+			"query", query.Name,
```

Comment:
Can we change `query` to `name` and `queryText` to `query` or `sql`?

Reply:
Do you feel comfortable with the changes to the headers here and throughout this file?